### PR TITLE
Fixups for github hosted templates

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

File: src/commands/init.ts [hunk 1]
```diff
@@ -49,22 +49,23 @@ export class InitCommand implements Command {
...
       env.registerStub(ElementGenerator, 'polymer-init-element:app');
       env.registerStub(ApplicationGenerator, 'polymer-init-application:app');
       let shopGenerator = createGithubGenerator({
-        owner: 'PolymerLabs',
-        repo: 'polykart',
+        owner: 'Polymer',
+        repo: 'shop',
       });
       env.registerStub(shopGenerator, 'polymer-init-shop:app');
       let prplGenerator = createGithubGenerator({
```
Comment (azakus) [inline src/commands/init.ts]: appDrawerTemplateGenerator?

File: src/init/github.ts [hunk 2]
```diff
@@ -59,7 +59,9 @@ export function createGithubGenerator(githubOptions: GithubGeneratorOptions) {
     }
 
     install() {
-      this.installDependencies();
+      this.installDependencies({
```
Comment (azakus) [inline src/init/github.ts]: Should this check for `package.json`

**Comment by azakus** [comment]
LGTM

File: src/init/github.ts (hunk 2 above)
Comment (justinfagnani) [inline src/init/github.ts]: eventually, yes. especially when a template needs it.

File: src/commands/init.ts (hunk 1 above)
Comment (justinfagnani) [inline src/commands/init.ts]: done

**Pull request merged**